WordPress/Classes/ViewRelated/Post/AbstractPostMenuHelper.swift

```diff
@@ -182,6 +194,8 @@ extension AbstractPostButton: AbstractPostMenuAction {
         static let draft = NSLocalizedString("posts.draft.actionTitle", value: "Move to draft", comment: "Label for an option that moves a post to the draft folder")
         static let delete = NSLocalizedString("posts.delete.actionTitle", value: "Delete permanently", comment: "Label for the delete post option. Tapping permanently deletes a post.")
         static let trash = NSLocalizedString("posts.trash.actionTitle", value: "Move to trash", comment: "Label for a option that moves a post to the trash folder")
+        // TODO: Replace with a namespaced string
+        static let retry = NSLocalizedString("Retry", comment: "User action to retry media upload.")
```

Comment:
This is a common key and I used it intentionally because there is presumably no time left to introduce new keys and get them translated for 25.1.